xtask/src/main.rs

```diff
@@ -117,7 +121,9 @@ fn run_action(workspace: Workspace, target: &str, extra_args: &[String]) {
         .wait()
         .unwrap();
 
-    if !status.success() {
+    // The clippy exception ensures that both workspaces are linted if one workspace fails. The
+    // exit status may be inaccurate, but we can live with this.
+    if !status.success() && (target != "clippy") {
```

Comment:
This will cause `cargo xtask clippy` on CI to never fail.

Reply:
We don't run clippy in CI, as we don't agree with all of the lints it suggests. Even if we were to build a clippy configuration, we'd probably find many of the lints subjective.